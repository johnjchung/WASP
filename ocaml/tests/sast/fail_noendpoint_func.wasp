Func int squareFloat () {
	list<int> a = [1,2,3,4,5];
	int i = a@3;
	return i;
}
