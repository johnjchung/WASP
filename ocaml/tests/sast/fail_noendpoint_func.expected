Fatal error: exception Symbol_table.NoEndpointError("YOU NEED TO DECLARE AT LEAST ONE ENDPOINT")
